net-im/signal-cli-bin/signal-cli-bin-0.7.4.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="signal-cli-${PV}"
+DESCRIPTION="Send and receive messages of Signal Messenger over a command line interface"
+HOMEPAGE="https://github.com/AsamK/signal-cli/wiki"
+SRC_URI="
+	https://github.com/AsamK/signal-cli/releases/download/v${PV}/${MY_P}.tar.gz -> ${P}.tar.gz
+	https://github.com/AsamK/signal-cli/raw/v${PV}/README.md -> ${P}.README.md
```

Comment:
I noticed the tarball includes *README.md* fine. You can use that and call `einstalldocs` in `src_install` to install it.

Reply:
I check that. Must be new with that release...

Reply:
cant find  a readme in signal-cli-bin-0.7.4.tar.gz